frame/dapps-staking/src/tests.rs

```diff
@@ -577,11 +596,21 @@ fn new_session_scenario_test() {
 
         advance_era();
         DappsStaking::on_finalize(0);
+        advance_session();
+
+        assert_ok!(DappsStaking::claim(Origin::signed(BOB_STASH), target_era));
 
         assert_eq!(Balances::free_balance(&BOB_STASH), 2_000 + 1); // +nomiante reward
         assert_eq!(Balances::free_balance(&BOB_CTRL), 20 + 0); // +0
-        assert_eq!(Balances::free_balance(&ALICE_STASH), 1_000 + 8); // +operator reward
+
+        assert_ok!(DappsStaking::claim(
+            Origin::signed(OPERATED_CONTRACT),
```

Comment:
It should be `Origin::signed(OPERATOR)`.

Reply:
modified it in 313ce01